### DISCUSSION TITLE
Conduct penetration test on Doris

### DISCUSSION THREAD

**Discussion by Quantity-kkk** [apache/doris#11540] [A - General / Q&A]
When I Conduct penetration test on Doris，I found the following problems：

- The 8030 port has a sourcemap file disclosure vulnerability. The specific address is [http://127.0.0.1:8030/main.0f5825b620e1b36ce46e.js.map]

- The 8060 port has database connection pool unauthorized access vulnerability. The specific address is [http://127.0.0.1:8060/connections]

- The 8040 port has device performance monitoring service unauthorized access vulnerability.The The specific address is [http://127.0.0.1:8040/]、[http://127.0.0.1:8040/mem_tracker]

Does anyone know how to fix these vulnerabilities?

渗透测试提示以下端口存在问题，有人有解决思路吗？
- 8030端口存在sourcemap文件泄露问题
- 8040端口存在设备性能监控服务未授权访问漏洞
- 8060端口存在数据库连接池未授权访问漏洞

**Comment by smallhibiscus** [comment]
Thanks for your test question, please add my WeChat and let's talk about the situation in detail. My WeChatID number is ```smallhibiscus```.
